datamonster_api/lib/company.py

```diff
@@ -26,6 +26,9 @@ def __repr__(self):
         ticker_str = "[{}] ".format(self.ticker) if self.ticker else ""
         return "<{}: {}{}>".format(self.__class__.__name__, ticker_str, self.name)
 
+    def __eq__(self, obj):
+        return isinstance(obj, Company) and self.get_details() == obj.get_details()
```

Comment:
Does it make sense to check equality against the ids (section_pks) instead?

Reply:
updated